rts/Lua/LuaVBOImpl.cpp

```diff
@@ -1059,6 +1218,18 @@ size_t LuaVBOImpl::InstanceDataFromFeatureIDs(const sol::stack_table& ids, int a
 	return InstanceDataFromImpl<CFeature>(ids, attrID, /*noop*/ 0u, elemOffsetOpt);
 }
 
+
```

Comment:
Hmm I forgot how it works. Will come back to this one later :)

Reply:
Similar to other `InstanceDataFrom*` but returns a 4x4 matrix.

Reply:
Ah yes yes, instead of sending an indirect index into a global GPU SSBO, it uploads the whole 4x4 matrix into a specified instance attribute(s).